lib/node_modules/@stdlib/string/base/replace-before-last/benchmark/benchmark.js

```diff
@@ -0,0 +1,58 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var isString = require( '@stdlib/assert/is-string' ).isPrimitive;
+var pkg = require( './../package.json' ).name;
+var replaceBeforeLast = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg, function benchmark( b ) {
+	var values;
+	var out;
+	var str;
+	var i;
+
+	str = 'To be, or not to be, that is the question.';
+	values = [
+		'foo',
+		'bar',
+		'beep',
+		'boop'
+	];
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		out = replaceBeforeLast( str, '.', values[ i%values.length ], str.length );
```

Comment:
```suggestion
		out = replaceBeforeLast( str, 'T', values[ i%values.length ], str.length );
```

Use the first character, as it represents the worst case scenario.